backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/appsearchmodal/internal/BpkSection.kt

```diff
@@ -76,11 +76,21 @@ internal fun BpkSectionItem(item: BpkItem, modifier: Modifier = Modifier) {
             icon = item.icon,
             contentDescription = null,
             size = BpkIconSize.Large,
-
         )
-        Column {
+        Column(
+            modifier = Modifier.weight(1f),
+        ) {
             BpkText(text = item.title)
-            BpkText(text = item.subtitle)
+            BpkText(
+                style = BpkTheme.typography.footnote,
+                text = item.subtitle,
+            )
+        }
+        item.tertiaryLabel?.let {
+            BpkText(
```

Comment:
the text colour should be text secondary here

Reply:
Updated also.